Bottom: c5d07194228e1c0c9f0da820c45faa22a6da0cb4
Top:    26fcaa26995fe8b04408d1568f23d3c4a0dcad71
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2013-01-01 19:25:16 +0000

Refresh of split-drc-routines-out-of-find

---

diff --git a/src/drc.c b/src/drc.c
index 4d50218..21332c6 100644
--- a/src/drc.c
+++ b/src/drc.c
@@ -30,36 +30,16 @@
 #include "config.h"
 #endif
 
-#include <stdio.h>
-#include <stdlib.h>
-#ifdef HAVE_STRING_H
-#include <string.h>
-#endif
-#include <math.h>
-#include <setjmp.h>
-#include <assert.h>
-
-#ifdef HAVE_SYS_TIMES_H
-#include <sys/times.h>
-#endif
-
-
 #include "global.h"
 
-#include "crosshair.h"
 #include "data.h"
 #include "draw.h"
 #include "error.h"
 #include "find.h"
-#include "mymem.h"
 #include "misc.h"
-#include "rtree.h"
 #include "polygon.h"
 #include "pcb-printf.h"
-#include "search.h"
-#include "set.h"
 #include "undo.h"
-#include "rats.h"
 
 #ifdef HAVE_LIBDMALLOC
 #include <dmalloc.h>
